### ISSUE TITLE
Make data easier to consume in bulk

### ISSUE THREAD

**Issue by fortuna** [ooni/backend#191]
I'll use this bug to collect ideas to make it easier to consume OONI data in bulk.

- [ ] **Move files under `<test_name>` directories (e.g. `autoclaved/jsonl/web_connectivity/...`)**
  This will allow one to fetch or process a test type without having to go over all the tests. One may list all files, filter, then use that as input. However thousands of filenames as input does not work well with big data tools.
Even if the `test_name` is under the date directory, that would already be useful (I could process one data at a time). Similarly, the filename prefix could have the test name (it doesn't help to have it in the middle).

- [ ] **Move files under `<country>` directories (e.g. `autoclaved/jsonl/web_connectivity/IT/...`)**
  This will allow one to fetch or process a test in a country without going over all the data. This would also be in a filename prefix.

- [ ] **Use a uniform file format with balanced number of measurements**. We can have a split file (e.g. `measurements-XXXXX-of-01000.jsonl.gz`) with about the same number of measurements each.
  The grouping by report is not useful to consumers. The mix of `json.lz4` and `tar.lz4` prevents effective use of big data tools. Furthermore, LZ4 is not well supported, which also prevents the use of many tools.

- [ ] **Move HTTP bodies to a separate datastore**.
  Most of the data is HTTP request/response body, which is often not useful for bulk processing. They would live in parallel files instead, maybe with the same index as the measurements to make joining easy (e.g. measurements-01234-of-10000 joins with bodies-01234-of-10000).

**Comment by FedericoCeratto** [comment]
Related to #203

**Comment by FedericoCeratto** [comment]
While implementing database backups (#766)  we discussed publishing the fastpath and JSONL tables to provide users with another way to access measurement data.

The jsonl table also provide indexing of the files on the S3 data bucket. This could be used to selectively download and reprocess only the most interesting measurement.

**Comment by hellais** [comment]
We now have better options for this and have mostly implemented the suggestions from @fortuna for the new data. 

I suggest any future work on this is done as part of https://github.com/ooni/data/issues/59. Closing.
